sherpa/astro/ui/tests/test_astro_ui_plot.py

```diff
@@ -1347,7 +1347,7 @@ def check_pha1_plot_model_component_plot():
     variants, as all we do is check the y axis range makes sense
     (as it wouldn't have with #1020 unfixed).
 
-    Any test calling this requires @requires_plotting
+    Any test calling this requires @requires_pylab
```

Comment:
When we come to adding a new plot backend we should check to see if we can refactor tests like this. Maybe they are really backend-specific (in which case we probably should move them to a separate test file) but maybe some can be abstracted. Anyway, not for this PR.